app/assets/stylesheets/membership_application.scss

```diff
@@ -36,3 +36,11 @@ body.applications.show {
     }
   }
 }
+
+.overdue-highlighted {
+  background-color: #FFA500;
+}
+
+.almost-due-highlighted {
+  background-color: yellow;
+}
```

Comment:
We have Bootstrap 3 in the app, so you can use  Bootstrap's existing background color definitions instead of creating your own. Docs for what's available are here: https://getbootstrap.com/docs/3.4/css/#helper-classes-backgrounds

You could use `bg-info` and `bg-danger`, for example.

Reply:
Used bg-info and bg-danger instead.